### DISCUSSION TITLE
su_y_my_pi questions

### DISCUSSION THREAD

**Discussion by siena-napoleon** [now-i-know-my-abcd/docs#80] [Where in the World is…? (Help)] [answered]
Hi, I'm doing research on extramedical pill use and have a DUC, but am confused about the structure/format of the data, and specifically the my_pi module. My questions are:

- Is it possible to get a copy of the full questionnaire (with scoring instructions) associated with this module? Or to at least know the name of the questionnaire associated with the "su_y_my_pi" data table?
- How is this questionnaire scored?
- When (at what visits) is this questionnaire administered, and to whom? This will help me figure out what my denominators should be...
- What do blank cells in the data mean? Do they always mean, for instance, that a question was not asked?

If anyone is willing or able to help me sort through this table in particular, I'd be SUPER grateful! I also have KSADS questions (about conduct disorder and antisocial personality disorder), but figured I'd start here...thanks!

**Comment by clarefmccann** [comment]
Hi Siena! Some of the questionnaires are in-house, meaning they are just questions ABCD wanted to ask and are not a part of a validated measure. If there is no scoring information in the data dictionary, this is probably the case for your questions of interest. As for the protocol, the best option is to download the data and see what time points are present if you can't find the actual questionnaire name and it's not listed in the study protocols [here](https://abcdstudy.org/scientists/protocols/). 

Feel free to post questions about KSADS as well, I hope this was helpful! 

Clare

**Comment by alexanderlavernewallace** [comment, marked as answer]
Hello!

The MYPI stands for the Mid-Year Phone Interview and represents the phone interviews that are conducted ~6 months between their annual visits. In this way, the MYPI is its own time point between the annual time points. As Clare said, the phone interviews are a set of follow up questions and are not necessarily a specific questionnaire with scoring. The MYPI is conducted with all participants regardless of whether they attended or missed their previous annual visit, so these measures are attempted with all participants still enrolled in the study. That said, any missing data means that these questions were not administered to the participant.

Hope this helps!

Alex
